tests/integration/docusaurus/connecting_to_your_data/database/snowflake_python_example.py

```diff
@@ -0,0 +1,91 @@
+import os
+
+from ruamel import yaml
+
+import great_expectations as ge
+
+from .util import load_data_into_database
+
+sfAccount = os.environ.get("SNOWFLAKE_ACCOUNT")
+sfUser = os.environ.get("SNOWFLAKE_USER")
+sfPswd = os.environ.get("SNOWFLAKE_PW")
+sfDatabase = os.environ.get("SNOWFLAKE_DATABASE")
+sfSchema = os.environ.get("SNOWFLAKE_SCHEMA")
+sfWarehouse = os.environ.get("SNOWFLAKE_WAREHOUSE")
+
+
+CONNECTION_STRING = f"snowflake://{sfUser}:{sfPswd}@{sfAccount}/{sfDatabase}/{sfSchema}?warehouse={sfWarehouse}"
+load_data_into_database(
+    "taxi_data",
+    "/Users/work/Development/great_expectations/integration/fixtures/data/reports/yellow_tripdata_sample_2019-01.csv",
+    CONNECTION_STRING,
+)
+
+context = ge.get_context()
+
+
+datasource_config = {
+    "name": "my_snowflake_datasource",
+    "class_name": "Datasource",
+    "execution_engine": {
+        "class_name": "SqlAlchemyExecutionEngine",
+        "connection_string": "snowflake://<USER_NAME>:<PASSWORD>@<ACCOUNT_NAME>/<DATABASE_NAME>/<SCHEMA_NAME>?warehouse=<WAREHOUSE_NAME>&role=<ROLE_NAME",
+    },
+    "data_connectors": {
+        "default_runtime_data_connector_name": {
+            "class_name": "RuntimeDataConnector",
+            "batch_identifiers": ["default_identifier_name"],
+        },
+        "default_inferred_data_connector_name": {
+            "class_name": "InferredAssetSqlDataConnector",
+            "name": "whole_table",
+        },
+    },
+}
+
```

Comment:
Please copy the comment from the postgres files here so when normal users view these scripts they will understand they don't need to do substitutions like this.